src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [titleFilter, setTitleFilter] = useState<string>('');
+  const [filter, setFilter] = useState<string>('');
+
+  const handleSelectedTodo = (todo: Todo) => {
+    const findTodo = todos.find(value => value === todo) as Todo;
+
+    setSelectedTodo(findTodo);
+  };
+
+  const handleModalClose = () => {
+    setSelectedTodo(null);
+  };
+
+  const handleFilter = (status: string) => {
+    if (status === 'all') {
```

Comment:
Move all cases to the enum.

Reply:
<img width="641" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/70220667/c89ad556-1bbb-44d3-8fba-75c8b20e2ffe">
still not fixed, create some enum for status values